dockers/docker-ptf/Dockerfile.j2

```diff
@@ -100,4 +100,4 @@ COPY ["supervisord.conf", "sshd.conf", "ptf_nn_agent.conf", "/etc/supervisor/con
 
```

Comment:
Currently, the conf.d file is only partial configuration, with the remaining (shared) configuration in /etc/supervisord.conf. Thus, I feel like the partial config file in conf.d should have a name specific to what it's configuring (along the lines of "container.conf"). The fact that it resides under /etc/supervisor/conf.d/ should be enough to should that it is modifying the supervisord configuration. See my comment below for more.